docs/pipeline.rst

```diff
@@ -760,11 +760,12 @@ ner_crf
           # Specifies the L2 regularization coefficient.
           L2_c: 0.1
 
-.. _section_pipeline_duckling:
+.. _ner_duckling_http:
 
-ner_duckling
-~~~~~~~~~~~~
-:Short: Adds duckling support to the pipeline to unify entity types (e.g. to retrieve common date / number formats)
+ner_duckling_http
+~~~~~~~~~~~~~~~~~
+:Short: Adds duckling support to the pipeline to unify
```

Comment:
not a change you introduced, but this sentence is super obscure. Let's say: Duckling lets you extract common entities like dates, amounts of money, distances, and others in a number of languages.